stdlib/public/SDK/Foundation/NSObject.swift

```diff
@@ -13,7 +13,10 @@
 @_exported import Foundation // Clang module
 import ObjectiveC
 
+// This exists to allow for dynamic dispatch on KVO methods added to NSObject.
+// Extending NSObject with these methods would disallow overrides.
```

Comment:
Annotating to save someone the time of figuring out the purpose of this protocol in the future.